brave-bin/PKGBUILD

```diff
@@ -55,7 +53,10 @@ package() {
 
 	install -Dm0755 "$pkgname.sh" "$pkgdir/usr/bin/brave"
 	install -Dm0644 -t "$pkgdir/usr/share/applications/" "brave-browser.desktop"
-	install -Dm0644 "logo.png" "$pkgdir/usr/share/pixmaps/brave-desktop.png"
+	# Install the icon using files in the archive
+	for i in 16x16 24x24 32x32 48x48 64x64 128x128 256x256; do
+		install -Dm644 "$pkgdir/usr/lib/$pkgname/product_logo_${i/x*/}.png" "$pkgdir/usr/share/icons/hicolor/$i/apps/brave-desktop.png"
```

Comment:
I wonder if `install` is the right tool for the job here. Does it serve any purpose copying them or should this be a move operation instead?

Reply:
From what I've read, `install` is basiclly `cp` and `chmod` at the same time. So I think it's the right tool

Reply:
Also parent directory creation. The other two bits are nice, I was jut thinking about the space aspect of leaving the extra copy behind. I guess since this is a `-bin` package we shouldn't be messing with the internals too much and can just leave the assets where it expects them and copy them for our own use.